wemake_python_styleguide/visitors/ast/subscripts.py

```diff
@@ -172,3 +172,43 @@ def _is_float_key(self, node: ast.expr) -> bool:
             isinstance(real_node, ast.Num) and
             isinstance(real_node.n, float)
         )
+
+
+@final
+class ConsecutiveSlicesVisitor(base.BaseNodeVisitor):
+    """Check the existance of consecutive slices."""
+
+    def __init__(self, *args, **kwargs) -> None:
+        """Create necessary variables."""
+        super().__init__(*args, **kwargs)
+        self._max_cons_slices: int = 0
+        self._first_oc: ast.Subscript
+
+    def visit_Subscript(self, node: ast.Subscript) -> None:
+        """
+        Visits subscript.
+
+        Raises:
+            ForbidConsecutiveSlicesViolation
+
+        """
+        self._check_consecutive(node)
+        self.generic_visit(node)
+
+    def _check_consecutive(self, node: ast.Subscript) -> None:
+        """Check if subscript node has a slice and a subscript."""
+        condition = (
```

Comment:
We can base this condition on `get_parent()`. Look, how it is structured:

```
(.venv) ~/Documents/github/wemake-python-styleguide  master ✔                        
» echo 'a[1:][2:]' > ex.py

(.venv) ~/Documents/github/wemake-python-styleguide  master ✔                        
» astboom ast < ex.py
Failed to read source from command line, trying to read it from STDIN:
========================================================================
======================================================================== 

<ast.Module>
  ├─ type_ignores: []
  └─ body
      └─ [0] <ast.Expr>
          ├─ lineno: 1
          ├─ col_offset: 0
          ├─ end_col_offset: 9
          ├─ end_lineno: 1
          └─ value: <ast.Subscript>
              ├─ lineno: 1
              ├─ col_offset: 0
              ├─ end_col_offset: 9
              ├─ end_lineno: 1
              ├─ ctx: <ast.Load>
              ├─ slice: <ast.Slice>
              │   ├─ lineno: 1
              │   ├─ col_offset: 6
              │   ├─ end_col_offset: 8
              │   ├─ end_lineno: 1
              │   ├─ step: None
              │   ├─ upper: None
              │   └─ lower: <ast.Constant>
              │       ├─ lineno: 1
              │       ├─ col_offset: 6
              │       ├─ end_col_offset: 7
              │       ├─ end_lineno: 1
              │       ├─ kind: None
              │       └─ value: 2
              └─ value: <ast.Subscript>
                  ├─ lineno: 1
                  ├─ col_offset: 0
                  ├─ end_col_offset: 5
                  ├─ end_lineno: 1
                  ├─ ctx: <ast.Load>
                  ├─ slice: <ast.Slice>
                  │   ├─ lineno: 1
                  │   ├─ col_offset: 2
                  │   ├─ end_col_offset: 4
                  │   ├─ end_lineno: 1
                  │   ├─ step: None
                  │   ├─ upper: None
                  │   └─ lower: <ast.Constant>
                  │       ├─ lineno: 1
                  │       ├─ col_offset: 2
                  │       ├─ end_col_offset: 3
                  │       ├─ end_lineno: 1
                  │       ├─ kind: None
                  │       └─ value: 1
                  └─ value: <ast.Name>
                      ├─ lineno: 1
                      ├─ col_offset: 0
                      ├─ end_col_offset: 1
                      ├─ end_lineno: 1
                      ├─ id: 'a'
                      └─ ctx: <ast.Load>

```

If we have nested subscripts with slices, then we can raise a violation.